index.html

```diff
@@ -3,16 +3,30 @@
   <head>
     <meta charset="UTF-8">
     <title>Fork Me! FCC: Test Suite Template</title>
-    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
 
   <body>
 
-<div class="block">
+<div class="container">
```

Comment:
- [x] Kindly use semantic approach instead of using div.
* It makes web pages more informative and adaptable
* For example: replace div by section at line 11
```suggestion 
<div class="container"> 
<section class="container">
```